blocks/header/header.js

```diff
@@ -355,13 +355,70 @@ function settingAltTextForSearchIcon() {
   searchImage.setAttribute('title', listOfAllPlaceholdersData.searchAltText);
 }
 
-async function fetchingPlaceholdersData() {
+async function fetchingPlaceholdersData(placeholdersData) {
   listOfAllPlaceholdersData = await fetchLanguagePlaceholders();
   const hamburger = document.querySelector('.nav-hamburger');
-  hamburger.setAttribute('title', listOfAllPlaceholdersData.hamburgerAltText);
+  hamburger.setAttribute('title', placeholdersData.hamburgerAltText);
   makeImageClickableNSettingAltText();
   settingAltTextForSearchIcon();
 }
+const toggleExpandLanguageSelector = (e) => {
+  const toggleContainer = e.currentTarget;
+  if (toggleContainer) {
+    const toggler = toggleContainer.querySelector('div.language-toggle');
+    const toggleContent = toggleContainer.querySelector('div.language-content');
+    if (e.type === 'click' && toggleContent) {
```

Comment:
Before this line, you can do `if (!toggler || !toggleContent) return;` so that you don't need to use that check everywhere below.

Reply:
Done